x-pack/solutions/security/plugins/kubernetes_security/public/hooks/use_eui_theme.ts

```diff
@@ -33,8 +33,8 @@ export const useEuiTheme = (...props: EuiThemeProps): EuiThemeReturn => {
 
     const extraEuiVars: ExtraEuiVars = {
       // eslint-disable-next-line @typescript-eslint/naming-convention
-      euiColorVis6_asText: shade(themeVars.euiColorVis6, 0.335),
-      buttonsBackgroundNormalDefaultPrimary: '#006DE4',
+      euiColorVis6_asText: shade(themeVars.euiColorVis6, 0.335), // TODO: replace with proper color token usage and remove shade
```

Comment:
k8s_security should be decommissioned in 9.0 and serverless, therefore not touching, but leaving a comment in case the dashboard will be revived in the future